src/NumberTheory/vinberg.jl

```diff
@@ -353,3 +353,163 @@ function vinberg_algorithm(S::ZZLat, upper_bound; v0=QQ[0;]::QQMatrix, root_leng
   return [r * B for r in roots]
 end
 
+
+function _get_tau(f::QQMatrix, Qb) ::QQBarFieldElem
+  tau = QQ(0)
```

Comment:
```suggestion
  tau = Qb(0)
```
or equivalently
```suggestion
  tau = QQBarFieldElem(0)
```
or
```suggestion
  tau = zero(Qb)
```
your choice ;-)